src/transform/convert/index.js

```diff
@@ -1,7 +1,74 @@
-import convertRecord from './common';
+import {MarcRecord} from '@natlibfi/marc-record';
+import moment from 'moment';
```

Comment:
Avoid using moment. Moment is deprecated https://momentjs.com/docs/#/-project-status/

Reply:
https://date-fns.org/ is okayish

Reply:
Migration from moment to non-deprecated dependency is done in unified manner for all existing conversions